Add migration script and guide for upgrading 2.0.4 to 2.1 (#13507)
kyma-project/kyma: docs/migration-guide-2.0.4-main.md

---
title: Migration Guide 2.0.4-main
---

> Once you upgrade from Kyma 2.0.4 to a newer version, either run the script 2.0.4-main-prom-upgrade-cleanup.sh found in /assets or perform the required patches from said script manually.

Reviewer: ```suggestion
When you upgrade from Kyma 2.0.4 to a newer version, either run the script `2.0.4-main-prom-upgrade-cleanup.sh` found in `/assets`, or perform the required patches from that script manually.
```

> When you upgrade from Kyma 2.0.4 to a newer version, either run the script `2.0.4-main-prom-upgrade-cleanup.sh` found in `/assets`, or perform the required patches from that script manually.

Reviewer: ```suggestion
When you upgrade from Kyma 2.0.4 to a newer version, take care of the Prometheus upgrade.
Either run the script `2.0.4-main-prom-upgrade-cleanup.sh` found in `/assets`, or perform the required patches from that script manually.
```
